server/src/main/java/org/hyperskill/database/serverside/JsonProtocol.java

```diff
@@ -0,0 +1,41 @@
+package org.hyperskill.database.serverside;
+
+import com.google.gson.JsonElement;
+import org.hyperskill.database.common.Request;
+import org.hyperskill.database.common.Response;
+
+import java.util.Objects;
+
+public class JsonProtocol {
+    JsonStorage storage = new JsonStorage();
```

Comment:
Would be nicer and cleaner to introduce inversion of control here by creating and using `Storage` interface instead of concrete implementation